boot/freeldr/CMakeLists.txt

```diff
@@ -3,3 +3,4 @@ add_subdirectory(bootsect)
 add_subdirectory(fdebug)
 add_subdirectory(freeldr)
 add_subdirectory(install)
+add_subdirectory(uefi)
```

Comment:
I can do that, I wanted to separate it on this directory since its entirely separated from the compilation happening from within the freeldr directory.
e.g both uefi, freeldr, install all compile an object.
Whatever you prefer